pretty.js

```diff
@@ -46,6 +46,13 @@ function filter (value) {
   return result
 }
 
+function isPinoLine (line) {
+  var requiredProperties = [ 'time', 'name', 'hostname', 'pid' ]
```

Comment:
There has to be a way to determine if the incoming line is a pino line. I have updated it to depend only on `hostname` and `pid` properties. They are used to build the `(pid on hostname):` prefix, which is not conditional as the code currently stands.